lib/services/beer.rb

```diff
@@ -5,14 +5,14 @@ class Beer
     THROTTLING_BEER_TIMEOUT = 60 * 5 # 5 minutes
 
     SCALE_OF_DRUNKNESS = {
-      1 => ['In Russia we say: mezdu pervoi i vtoroi pererivchik nebolshoi', '*Good start!*'],
-      2 => ['️*While you were finishing this beer - I was downd my third glass*', '*I hope you got here by bus?*'],
-      3 => ['*Now it is time to pee*', '*Time to discuss the last GOT*'],
-      4 => ['*The good, the bad and the fourth glass*', '*What do you think about karaoke bar?*'],
-      5 => ['*How many fingers are there?*', '*Time to call your ex*'],
-      6 => ['*Friendly reminder: in Saint-Petersburg the bridges are rised at night*', '*Are you ok, buddy?*'],
-      7 => ['*Remember your first glass? Neither do I*', '*I think it is time to get hard stuff*'],
-      8 => ['*Now I leave your alone with your beer, YOU WON!*']
+      1 => ['🇷🇺 In Russia we say: mezdu pervoi i vtoroi pererivchik nebolshoi 🇷🇺', '💫 Good start! 💫'],
+      2 => ['️🏋️‍♂️ While you were finishing this beer - I was downd my third glass 🏋️‍♀️', '👮‍♂️ I hope you got here by bus? 👮‍♂️'],
+      3 => ['🐳 Now it is time to pee 🐳', '🤓 Time to discuss the last GOT 🤓'],
+      4 => ['4️⃣ The good, the bad and the fourth glass 4️⃣', '🎙 What do you think about karaoke bar? 🎙'],
+      5 => ['🤞 How many fingers are there? ✌️', '💔 Time to call your ex 💔'],
+      6 => ['⌛ Friendly reminder: in Saint-Petersburg the bridges are rised at night ⌛', '🥴 Are you ok, buddy? 🥴'],
```

Comment:
Bridges are an important thing, perhaps it would be better to left only them in this array